Description: Fix multimediakit examples link failure (undefined reference to pulseaudio functions)

QT_CFLAGS_PULSEAUDIO and QT_LIBS_PULSEAUDIO are defined by Qt's configure.
Qt Mobility's confgure should defined them.
As a workaround, define a 'standard' value.

Author: Fathi Boudra <fabo@debian.org>

--- a/src/multimedia/effects/effects.pri
+++ b/src/multimedia/effects/effects.pri
@@ -6,8 +6,8 @@ unix:!mac:!symbian {
         DEFINES += QT_MULTIMEDIA_PULSEAUDIO
         PRIVATE_HEADERS += effects/qsoundeffect_pulse_p.h
         SOURCES += effects/qsoundeffect_pulse_p.cpp
-        QMAKE_CXXFLAGS += $$QT_CFLAGS_PULSEAUDIO
-        LIBS += $$QT_LIBS_PULSEAUDIO
+        QMAKE_CXXFLAGS += -D_REENTRANT
+        LIBS += -lpulse
     } else {
         DEFINES += QT_MULTIMEDIA_QMEDIAPLAYER
         PRIVATE_HEADERS += effects/qsoundeffect_qmedia_p.h
